### ISSUE TITLE
Inaccessible tooltips

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3121]
Several controls are followed by info icons which show a tooltip with additional information on mouse hover. These tooltips can only be triggered by mouse users and are not available for keyboard and screen reader users. Additionally, the current tooltip interaction may be problematic for screen magnification users, since: 
- there is no mechanism available to dismiss the additional content without moving  the pointer hover; and 
- the pointer cannot be moved over the additional content without causing it to  disappear.

**Suggested Solution** 
Implement the info icons as buttons, display the tooltips on focus and on hover, and use the aria-described by attribute on the button referencing the ID of the tooltip container to announce it for screen reader users. This approach has the disadvantage that the tooltip text cannot be triggered on touch devices and with some speech input software. Additionally ensure that the tooltips can be dismissed by pressing Escape and that they are hoverable. 

For more information on implementing accessible tooltips, refer to https://inclusive components.design/tooltips-toggletips/. 
There are a few alternatives to the suggested remediation path, e.g.: 
- Implement the info buttons as "toggletips" which require activation to reveal or hide  the additional information. 
- Ensure that help is always available in the label text by including more detail in the  label wherever possible and providing a "toggletip" with the remaining information. 
- Provide a link to a documentation page with explanatory notes for customization options. 

Severity: 1

**Comment by engbergandreas** [comment]
I kinda feel like this has been addressed with https://github.com/OpenSpace/OpenSpace/commit/e3f5b83f7476c8507dccd45ab98e4731504af532?  @ylvaselling  @WeirdRubberDuck

**Comment by WeirdRubberDuck** [comment]
I think we still need to verify if the text in the tooltip is accessible for screen-readers

**Comment by WeirdRubberDuck** [comment]
The info icons are now implemented as "toggle-tips", as suggested in this issue. I've also verified that the ifo is communicated to the screen readers

After adding that the tooltips are triggered on focus, as suggested from Mantine: https://mantine.dev/core/tooltip/#accessibility, this is closed
